include/variables.h

```diff
@@ -4,7 +4,7 @@
 #include "z64.h"
 #include "segment_symbols.h"
 
-extern Mtx D_01000000;
+extern Mtx gSoftSpriteMartixStatic;
```

Comment:
```suggestion
extern Mtx gSoftSpriteMatrixStatic;
```
you had one job Sadge